sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/TestSuiteBase.java

```diff
@@ -469,8 +469,8 @@ public static CosmosItemProperties createDocument(CosmosAsyncContainer cosmosCon
     }
```

Comment:
Should consider adding tests for `TracerProvider`.

Reply:
Also, consider adding a nesting call example to validate the behavior of spans.

Reply:
This is cover by start span count assert , test will fail if there is nesting . Will add explicit test in next pr covering event